### PR TITLE
Misc doc fixes

### PR DESCRIPTION
**Pull request by ThibsG**

### CODE REVIEW

File: documentation/docs/develop.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
-One of the advantages of using `mse` to protect your application and your data in the cloud, is that you don't need to adapt your own Python application. Indeed, you just need to pick your original code, design a standard Flask application without specific intructions, write a configuration TOML file and run the `deploy` subcommand. 
+One of the advantages of using MSE to protect your application and your data in the cloud, is that your Python application does not need to be adapted.
```
Comment (ccorsin) [inline documentation/docs/develop.md]: adapted to what? `modified to fit some specific constrains' for example?

File: documentation/docs/develop.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+Indeed, you just need to pick your original code, design a standard Flask application without any specific intruction, write a configuration TOML file and run the `deploy` subcommand.
```
Comment (ccorsin) [inline documentation/docs/develop.md]: Indeed, you just need -> You simply need

File: documentation/docs/develop.md [hunk 3]
```diff
@@ -13,13 +14,15 @@ Before sending the Python code of your microservice, each file is encrypted but:
...
 
-This code is supposed to be sharable, as your convenience, to any users in order to check the trustworthiness of your app. As a matter of fact, *do not write any secret into your code*. For example: passwords or keys to connect to a third-party service like a remote storage or a database. 
+This code is supposed to be sharable, as your convenience, to any user in order to check the trustworthiness of your app.
+As a matter of fact, *do not write any secret into your code*.
+For example: passwords or keys to connect to a third-party service like a remote storage or a database.
 
-If you need such secrets to run your code, you can write a `secrets.json` file. Please see the example below. This file will be sent to the enclave after the latter has been verified during the app deployment. Your application will then be able to read it to retrieve the secrets it needs.
+If you need such secrets to run your code, write them in a `secrets.json` file. Please see the example below.
+This file will be sent to the enclave after the latter has been verified during the app deployment.
+Your application will then be able to read it to retrieve the secrets it needs.
```
Comment (ccorsin) [inline documentation/docs/develop.md:24]: No need to specify that this file is encrypted somewhere ?

File: documentation/docs/develop.md [hunk 4]
```diff
@@ -47,6 +50,14 @@ def whoami():
     return secrets["login"]
 ```
 
+!!! info "Encrypting some secrets"
+
+    If your application requires some secrets to be hidden from the SGX operator, write those secrets in another file, for example `secrets_to_seal.json`.
+    Then you can seal this `secrets_to_seal.json` file with the command `msehome seal`.
```
Comment (ccorsin) [inline documentation/docs/develop.md]: ...with the `msehome seal`command.

File: documentation/docs/develop.md [hunk 5]
```diff
@@ -110,9 +121,9 @@ A default `.mseignore` is generated by the `mse scaffold` command.
 
 When you declare the memory size through the field `hardware` in the `mse.toml`, you shall consider that a part of this memory is used by the system itself. 
 
-All the librairies needed to run your application will be loaded in that memory. Therefore, the effective memory size available for your application is about: `hardware_memory - librairies_size`. 
+All the libraries needed to run your application will be loaded in that memory. Therefore, the effective memory size available for your application is about: `hardware_memory - librairies_size`. 
 
-When calling the docker on your localhost, you can use the option `--memory` to estimate your effective memory size. See our [github](https://github.com/Cosmian/mse-docker-base#determine-the-enclave-memory-size-of-your-image) for more details.
+When calling the docker on your localhost, you can use the option `--memory` to estimate your effective memory size. See our [MSE Docker base README](https://github.com/Cosmian/mse-docker-base#determine-the-enclave-memory-size-of-your-image) for more details.
```
Comment (ccorsin) [inline documentation/docs/develop.md]: the Docker ? -> the container ? the Docker service ?

File: documentation/docs/flow.md [hunk 6]
```diff
@@ -1,15 +1,15 @@
-In the mse home usage scenario, two roles are defined:
+In the MSE Home usage scenario, two roles are defined:
```
Comment (ccorsin) [inline documentation/docs/flow.md]: two seperate roles

File: documentation/docs/flow.md [hunk 7]
```diff
@@ -1,15 +1,15 @@
...
 
 - The code provider. This actor writes and owns the code of the microservice. It could also be the recipient of the result of this application.
-- The sgx operator. This actor owns the sgx hardware which runs the code and the data to run against the mse application.
+- The SGX operator. This actor owns the SGX hardware which runs the code and the data to run against the MSE application.
 
-MSE home deals with several trust issues. Noone needs to trust in anyone:
+MSE Home deals with several trust issues. No one needs to trust in anyone:
```
Comment (ccorsin) [inline documentation/docs/flow.md]: no one has to trust anyone

File: documentation/docs/flow.md [hunk 8]
```diff
@@ -1,15 +1,15 @@
...
 
-- The code provider can send the code in plaintext or encrypted to the sgx operator. Only the SGX enclave is able to decrypt and run it.
-- The code provider can send secrets required by the code in plaintext or encrypted to the sgx operator. No one can alter the result before the code provider gets them.
+- The code provider can send the code in plaintext or encrypted to the SGX operator. Only the SGX enclave is able to decrypt and run it.
```
Comment (ccorsin) [inline documentation/docs/flow.md]: The code provider can send plaintext or encrypted code to the SGX operator

File: documentation/docs/flow.md [hunk 9]
```diff
@@ -1,15 +1,15 @@
...
+- The code provider can send secrets required by the code in plaintext or encrypted to the SGX operator. No one can alter the result before the code provider decrypts it.
 - The result of the code can be generated in plaintext or encrypted and readable only by the code provider. Only the code provider could decrypt it.
-- The code runs on the SGX technology so the memory is fully encrypted and the code integrity is verifiable at any times. Noone can access the data, the result or the code during execution.
+- The code runs on the SGX technology so the memory is fully encrypted and the code integrity is verifiable at any time. No one can access the data, the result or the code during execution.
```
Comment (ccorsin) [inline documentation/docs/flow.md]: The code runs on the SGX technology _where_ the memory

File: documentation/docs/flow.md [hunk 10]
```diff
@@ -1,15 +1,15 @@
...
 
-The following flow sums up the chained step from to deploy and run the code by being compliant with the previous privacy & security levels
+The following flow sums up the chained steps to deploy and run the code by being compliant with the previous privacy & security levels
```
Comment (ccorsin) [inline documentation/docs/flow.md]: ...run code, while being compliant...

File: documentation/docs/getting_started.md [hunk 11]
```diff
@@ -231,59 +260,112 @@ The file `workspace/sgx_operator/evidence.json` and the previous file `workspace
...
+    From a user perspective, this is exactly the same as fetching `/result/secrets` endpoint.
+    Under the hoods, the original JSON file `secrets_to_seal.json` is transfered
+    sealed to the enclave (see how to [seal secrets](#seal-your-secrets)).
+
+    When [starting](#finalize-the-configuration-and-run-the-application), 
+    the app seamlessly decrypts this file with the enclave's private key, 
+    as sealed secrets are encrypted using the enclave's public key.
+    Data from `secrets_to_seal.json` is then accessible from the Flask app, through `SEALED_SECRETS_PATH` env variable.
+
+    This is the way to protect secrets from the SGX operator.
```
Comment (ccorsin) [inline documentation/docs/getting_started.md:368]: This is how you can protect secrets from...

File: documentation/docs/getting_started.md [hunk 12]
```diff
@@ -231,59 +260,112 @@ The file `workspace/sgx_operator/evidence.json` and the previous file `workspace
...
+
+
+First, the SGX operator collects the result (which is encrypted):
```
Comment (ccorsin) [inline documentation/docs/getting_started.md]: ...the result (encrypted)

File: documentation/docs/develop.md (hunk 1 above)
Comment (ThibsG) [reply documentation/docs/develop.md]:
```suggestion
One of the advantages of using MSE to protect your application and your data in the cloud, is that your original Python application does not need to be modified.
```

File: documentation/docs/develop.md (hunk 2 above)
Comment (ThibsG) [reply documentation/docs/develop.md]:
```suggestion
You simply need to pick your original code, design a standard Flask application without any specific intruction, write a configuration TOML file and run the `deploy` subcommand.
```

File: documentation/docs/develop.md [hunk 13]
```diff
@@ -13,13 +14,15 @@ Before sending the Python code of your microservice, each file is encrypted but:
...
 - `requirements.txt`
 
-This code is supposed to be sharable, as your convenience, to any users in order to check the trustworthiness of your app. As a matter of fact, *do not write any secret into your code*. For example: passwords or keys to connect to a third-party service like a remote storage or a database. 
+This code is supposed to be sharable, as your convenience, to any user in order to check the trustworthiness of your app.
+As a matter of fact, *do not write any secret into your code*.
+For example: passwords or keys to connect to a third-party service like a remote storage or a database.
 
-If you need such secrets to run your code, you can write a `secrets.json` file. Please see the example below. This file will be sent to the enclave after the latter has been verified during the app deployment. Your application will then be able to read it to retrieve the secrets it needs.
+If you need such secrets to run your code, write them in a `secrets.json` file. Please see the example below.
+This file will be sent to the enclave after the latter has been verified during the app deployment.
```
Comment (ThibsG) [inline documentation/docs/develop.md:22]:
```suggestion
This file will be sent to the enclave after the latter has been verified during the app deployment.
Note that this file is not encrypted and can be read by the SGX operator.
```

File: documentation/docs/develop.md (hunk 3 above)
Comment (ThibsG) [reply documentation/docs/develop.md:24]: I suggested a modification by adding a line right before this one, explaining the file is not encrypted.

File: documentation/docs/develop.md (hunk 4 above)
Comment (ThibsG) [reply documentation/docs/develop.md]:
```suggestion
    Then you can seal this `secrets_to_seal.json` file with the `msehome seal` command.
```

File: documentation/docs/flow.md (hunk 6 above)
Comment (ThibsG) [reply documentation/docs/flow.md]:
```suggestion
In the MSE Home usage scenario, two separate roles are defined:
```

File: documentation/docs/flow.md (hunk 7 above)
Comment (ThibsG) [reply documentation/docs/flow.md]:
```suggestion
MSE Home deals with several trust issues. No one has to trust anyone:
```

File: documentation/docs/flow.md (hunk 8 above)
Comment (ThibsG) [reply documentation/docs/flow.md]:
```suggestion
- The code provider can send plaintext or encrypted code to the SGX operator. Only the SGX enclave is able to decrypt and run it.
```

File: documentation/docs/flow.md (hunk 9 above)
Comment (ThibsG) [reply documentation/docs/flow.md]:
```suggestion
- The code runs on the SGX technology where the memory is fully encrypted and the code integrity is verifiable at any time. No one can access the data, the result or the code during execution.
```

File: documentation/docs/flow.md (hunk 10 above)
Comment (ThibsG) [reply documentation/docs/flow.md]:
```suggestion
The following flow sums up the chained steps to deploy and run the code while being compliant with the previous privacy & security levels
```

File: documentation/docs/getting_started.md (hunk 11 above)
Comment (ThibsG) [reply documentation/docs/getting_started.md:368]: Sorry but this is an easter egg

File: documentation/docs/getting_started.md (hunk 12 above)
Comment (ThibsG) [reply documentation/docs/getting_started.md]:
```suggestion
First, the SGX operator collects the encrypted result:
```

File: documentation/docs/develop.md [hunk 14]
```diff
@@ -110,9 +121,9 @@ A default `.mseignore` is generated by the `mse scaffold` command.
 
 When you declare the memory size through the field `hardware` in the `mse.toml`, you shall consider that a part of this memory is used by the system itself. 
 
-All the librairies needed to run your application will be loaded in that memory. Therefore, the effective memory size available for your application is about: `hardware_memory - librairies_size`. 
+All the libraries needed to run your application will be loaded in that memory. Therefore, the effective memory size available for your application is about: `hardware_memory - librairies_size`.
```
Comment (ThibsG) [inline documentation/docs/develop.md]:
```suggestion
All the libraries needed to run your application will be loaded in that memory. Therefore, the effective memory size available for your application is about: `hardware_memory - libraries_size`. 
```

File: documentation/docs/develop.md (hunk 5 above)
Comment (ThibsG) [reply documentation/docs/develop.md]:
```suggestion
When running the Docker container locally, you can use the option `--memory` to estimate your effective memory size. See our [github](https://github.com/Cosmian/mse-docker-base#determine-the-enclave-memory-size-of-your-image) for more details.
```

**Pull request merged**